cmd/kube-proxy/app/server_windows.go

```diff
@@ -79,7 +79,11 @@ func (s *ProxyServer) platformCheckSupported() (ipv4Supported, ipv6Supported, du
 }
 
 // createProxier creates the proxy.Provider
-func (s *ProxyServer) createProxier(config *proxyconfigapi.KubeProxyConfiguration, dualStackMode bool) (proxy.Provider, error) {
+func (s *ProxyServer) createProxier(config *proxyconfigapi.KubeProxyConfiguration, dualStackMode, initOnly bool) (proxy.Provider, error) {
+	if initOnly {
+		// --init-only is not implemented on Windows
+		return nil, nil
```

Comment:
Should that be an error then rather than `nil`?

I guess that depends on whether it's possible to run kube-proxy in winkernel mode in a non-privileged pod, which I don't know. (Does windows kubernetes even have any equivalent of `capabilities`?)

Reply:
@jayunit100 @jsturtevant can you help us to answer these questions?

Reply:
kube-proxy on windows needs to be run in privileged mode. either an error or log statement would be helpful here if someone specified this field. As it is now it would be confusing or lead to expectation that it could work as it silently runs.

Reply:
> Should that be an error then rather than nil?

No, the init can go wrong and return an error. But if it's ok it should return err==nil. But the returned proxier is nil to indicate that only initialization is done.

Reply:
Ah, you mean on Windows (missed that). Then, yes it should. I will update.